tests/simulator/test_advance.py

```diff
@@ -96,7 +99,8 @@ def vthz(x):
                                       "density": density,
                                       "vbulkx": vx, "vbulky": vy, "vbulkz": vz,
                                       "vthx": vthx, "vthy": vthy, "vthz": vthz,
-                                      "nbr_part_per_cell": nbr_part_per_cell})
+                                      "nbr_part_per_cell": nbr_part_per_cell,
+                                      "init": {"seed": 1337}})
```

Comment:
do we want to keep the seed? I think it's better without no? (more chances to see a bug?)

Reply:
will remove